Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/BaseTeleOp.java

```diff
@@ -0,0 +1,59 @@
+package org.firstinspires.ftc.team6220_PowerPlay;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.Stack;
+import java.util.logging.Filter;
+
+abstract public class BaseTeleOp extends BaseOpMode {
+    public Float[] GamePadInputAVG() {
```

Comment:
This method needs a design document.  Would you please write what this code tries to do, and how it does that?